charts/incubator/alfresco-dbp/requirements.yaml

```diff
@@ -3,15 +3,11 @@ dependencies:
     version: 3.0.0
     condition: alfresco-content-services.enabled
     repository: https://kubernetes-charts.alfresco.com/incubator
-  - name: alfresco-sync-service
```

Comment:
We'll still need to pull this in until it's part of the ACS chart.

Reply:
Sync service is part of the ACS chart now, for both charts of ACS 6.1.1 and ACS 6.2.0